.github/workflows/ci.yml

```diff
@@ -84,14 +84,9 @@ jobs:
         run: |
           nox -s test-${{ matrix.python-version }}
       - name: Upload Coverage
-        uses: codecov/codecov-action@v1
-        with:
-          token: 3bb38a90-137e-4efe-9627-ca66de472c13
-          file: ./coverage.xml
-          flags: unittests
-          name: codecov-umbrella
-          yml: ./codecov.yml
-          fail_ci_if_error: true
+        run: ./_travis/upload_coverage.sh
```

Comment:
Just a suggestion, we should put our CI script like `upload_coverage.sh` in a folder maybe named ci.

Reply:
@RatanShreshtha I renamed _travis/ to ci/, please take another look